wa/framework/configuration/core.py

```diff
@@ -891,16 +891,16 @@ class JobSpec(Configuration):
                            Similar to IDs but do not have the uniqueness restriction.
                            If specified, labels will be used by some result
                            processes instead of (or in addition to) the workload
-                           name. For example, the csv result processor will put
+                           name. For example, the csv output processor will put
                            the label in the "workload" column of the CSV file.
                            '''),
         ConfigurationPoint('augmentations', kind=toggle_set, merge=True,
                            aliases=["instruments", "processors", "instrumentation",
-                                    "result_processors", "augment"],
+                                    "output_processors", "augment"],
```

Comment:
Should keep the `"result_processors"` alias as well, for compatibility.

Reply:
Fixed